src/app/main.ts

```diff
@@ -33,6 +33,10 @@
     jQuery(document).on('renuo-cms-reload', function () {
       controller.init();
     });
+
+    jQuery(document).on('renuo-cms-reload', function (event:Event, htmlContentBlocks:HTMLElement[]) {
```

Comment:
Ahh ok, Simon said it should be possible to reload multiple and that they should already be htmlElements. Maybe you should discuss what you really need, because I don't know it...